packages/eslint-config/test/__snapshots__/typescript-eslint.spec.js.snap

```diff
@@ -121,9 +133,6 @@ Object {
     "@typescript-eslint/no-duplicate-enum-values": Array [
       "error",
     ],
-    "@typescript-eslint/no-duplicate-imports": Array [
-      "error",
-    ],
```

Comment:
They deprecated this rule in favor of `import/no-duplicates`: https://github.com/import-js/eslint-plugin-import/blob/HEAD/docs/rules/no-duplicates.md

We already have that enabled.